### PR TITLE
Implemented New Home Screen

### LINKED ISSUE #116: Improved Home Screen
**Actual Behaviour**
Here is the current Home Screen Screenshot:
Screenshot_20190329-182455

**Expected Behaviour**
New and Minimal Home Screen with Cards:
Home Screen Device connected

**Would you like to work on the issue?**
Yes, I would like to work on this issue 👍 

@CloudyPadmal @iamareebjamal @mariobehling

### PR DESCRIPTION
**Pull request by vishal-sehgal**
Fixes #116 

**Changes**: 
New Home Screen Layout.

**Screenshot/s for the changes**: 

**Checklist**: 
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**
[#116 Demo Apk.zip](https://github.com/fossasia/neurolab-android/files/3025319/116.Demo.Apk.zip)

### CODE REVIEW

**Comment by vishal-sehgal** [comment]
@iamareebjamal @CloudyPadmal @jddeep Please review this PR :)

**Comment by jddeep** [comment]
@CoderVishalSehgal the PR changes look fair. I request @iamareebjamal to take a look on the same.
@CoderVishalSehgal But do squash your commits into one. Cheers!

**Comment by vishal-sehgal** [comment]
@jddeep Thanks for reviewing! I will squash them very soon.

File: app/build.gradle [hunk 1]
```diff
@@ -33,9 +33,12 @@ dependencies {
...
     implementation "com.android.support:support-v4:$rootProject.supportLibraryVersion"
     implementation "com.android.support.constraint:constraint-layout:$rootProject.constraintLayoutVersion"
+    implementation "com.android.support:cardview-v7:$rootProject.supportLibraryVersion"
     // Test
     implementation 'com.android.support.constraint:constraint-layout:1.1.3'
     testImplementation "junit:junit:$rootProject.junitVersion"
     androidTestImplementation "com.android.support.test:runner:$rootProject.testRunnerRulesVersion"
     androidTestImplementation "com.android.support.test.espresso:espresso-core:$rootProject.espressoVersion"
+
+
```
Comment (iamareebjamal) [inline app/build.gradle]: Unnecessary changes

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -26,4 +26,34 @@
...
+    <dimen name="margin_seperator_vertical">8dp</dimen>
+    <dimen name="btn_vj_margin_top">16dp</dimen>
+    <dimen name="btn_vj_margin_right_end">32dp</dimen>
+    <dimen name="btn_serial_margin_start_left">32dp</dimen>
+    <dimen name="btn_serial_margin_top">16dp</dimen>
+    <dimen name="btn_focus_margin_end_right">32dp</dimen>
+    <dimen name="btn_focus_margin_bottom">16dp</dimen>
+    <dimen name="btn_relax_margin_start_left">32dp</dimen>
+    <dimen name="btn_relax_margin_bottom">16dp</dimen>
+    <dimen name="margin_bottom_device_status_textview">8dp</dimen>
```
Comment (iamareebjamal) [inline app/src/main/res/values/dimens.xml]: Reuse margin from standard set

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -34,4 +34,4 @@
         <item name="android:textColor">#000</item>
         <item name="android:textSize">19sp</item>
     </style>
-</resources>
+</resources>
```
Comment (iamareebjamal) [inline app/src/main/res/values/styles.xml]: Revert

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Remove splash screen png

File: app/build.gradle (hunk 1 above)
Comment (vishal-sehgal) [reply app/build.gradle]: Done 👍

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/styles.xml]: Done 👍

**Comment by vishal-sehgal** [comment]
@iamareebjamal Removed _bg_splash_screen.png_ 👍

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/dimens.xml]: This is also done 👍

File: app/src/main/res/values/colors.xml [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
+    <color name="colorPrimary">#4c5b66</color>
+    <color name="colorPrimaryDark">#23323c</color>
     <color name="colorAccent">#009688</color>
     <color name="colorControlHighlight">#ffffff</color>
     <color name="launcher_btn_txt_clr">#ffffff</color>
     <color name="color_view_background">#dad</color>
     <color name="color_white_background">#fff</color>
+    <color name="white">#FFFFFF</color>
```
Comment (iamareebjamal) [inline app/src/main/res/values/colors.xml]: Use android:colors/white

File: app/src/main/res/values/colors.xml [hunk 5]
```diff
@@ -1,10 +1,14 @@
...
+    <color name="home_activity_background">#F2F2F2</color>
+    <color name="black">#000000</color>
```
Comment (iamareebjamal) [inline app/src/main/res/values/colors.xml]: Same

File: app/src/main/res/values/dimens.xml [hunk 6]
```diff
@@ -26,4 +26,27 @@
...
+    <dimen name="margin_top_home_checkboxes">8dp</dimen>
+    <dimen name="home_card_two_elevation">4dp</dimen>
+    <dimen name="margin_set_up_title">8dp</dimen>
+    <dimen name="margin_top_set_up_title">24dp</dimen>
+    <dimen name="height_seperator_horizontal">1dp</dimen>
+    <dimen name="width_seperator_vertical">1dp</dimen>
+    <dimen name="height_seperator_vertical">150dp</dimen>
+    <dimen name="margin_seperator_vertical">8dp</dimen>
+    <dimen name="btn_vj_margin_top">16dp</dimen>
+    <dimen name="btn_vj_margin_right_end">32dp</dimen>
```
Comment (iamareebjamal) [inline app/src/main/res/values/dimens.xml]: Same, use standard set of dimens, no need to duplicate

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (iamareebjamal) [reply app/src/main/res/values/styles.xml]: Not done

File: app/src/main/res/values/dimens.xml (hunk 6 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/dimens.xml]: These are the dimens value for different attributes and they can have duplicate values, Suppose in future if someone changes the value of particular **_dimen_** then it will affect other uses too.

File: app/src/main/res/values/colors.xml (hunk 5 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/colors.xml]: Done

File: app/src/main/res/values/colors.xml (hunk 4 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/colors.xml]: done

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/styles.xml]: I changed this but after pressing Alt+Ctrl+L it reformats the code. But this time I revert it.

**Comment by vishal-sehgal** [comment]
@iamareebjamal  I fixed all the issues, please review.

File: app/src/main/res/values/dimens.xml (hunk 6 above)
Comment (iamareebjamal) [reply app/src/main/res/values/dimens.xml]: Then in future, we'll add a separate dimen

Comment (vishal-sehgal) [reply app/src/main/res/values/dimens.xml]: Okay, now what to do, May I know Why aren't you merging the PR then?

Comment (iamareebjamal) [reply app/src/main/res/values/dimens.xml]: Because "Same, use standard set of dimens, no need to duplicate"

File: app/src/main/res/values/dimens.xml [hunk 7]
```diff
@@ -23,7 +23,17 @@
     <dimen name="cbs_text_view_title_margin">6dp</dimen>
     <dimen name="launcher_title_text_size">16sp</dimen>
     <dimen name="btns_text_view_title_margin">4dp</dimen>
-    <dimen name="child_program_linear_layout_margin">8dp</dimen>
     <dimen name="launcher_btn_height">80dp</dimen>
     <dimen name="launcher_btn_width">0dp</dimen>
+    <dimen name="checkbox_margin_top">8dp</dimen>
```
Comment (iamareebjamal) [inline app/src/main/res/values/dimens.xml]: Duplicate

File: app/src/main/res/values/dimens.xml [hunk 8]
```diff
@@ -23,7 +23,17 @@
...
+    <dimen name="device_status_icon_width">10dp</dimen>
+    <dimen name="match_constraint_width">0dp</dimen>
+    <dimen name="home_card_one_corner_radius">4dp</dimen>
```
Comment (iamareebjamal) [inline app/src/main/res/values/dimens.xml]: Duplicate

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (iamareebjamal) [reply app/src/main/res/values/styles.xml]: Still not reverted

File: app/src/main/res/values/dimens.xml (hunk 7 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/dimens.xml]: Can you please elaborate more, what is duplicate here?

File: app/src/main/res/values/dimens.xml (hunk 8 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/dimens.xml]: Can you please elaborate more, what is duplicate here, I'm not getting you?

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/styles.xml]: Screenshot 2019-04-03 at 3 22 10 PM

**Comment by vishal-sehgal** [comment]
@iamareebjamal @CloudyPadmal  Can you please explain me clearly, What's wrong now?

File: app/src/main/res/values/dimens.xml (hunk 7 above)
Comment (iamareebjamal) [reply app/src/main/res/values/dimens.xml]: 8dp is already defined, use that

**Comment by vishal-sehgal** [comment]
@iamareebjamal Can you please take a look again, I fixed requested changes.

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (iamareebjamal) [reply app/src/main/res/values/styles.xml]: This file should not show in File Changes, that's it

File: app/src/main/res/layout/content_main.xml [hunk 9]
```diff
@@ -1,165 +1,244 @@
...
-
-        <LinearLayout
-            android:id="@+id/settings_linear_layout"
+    android:background="@color/home_activity_background"
+    app:layout_behavior="@string/appbar_scrolling_view_behavior">
+
+
+    <android.support.v7.widget.CardView
+        android:id="@+id/cardView"
+        android:layout_width="@dimen/launcher_btn_width"
```
Comment (iamareebjamal) [inline app/src/main/res/layout/content_main.xml]: 0dp -> spacing_none

**Comment by iamareebjamal** [comment]
Please take a look at how dimens standard sets are supposed to look like https://github.com/fossasia/open-event-android/blob/development/app/src/main/res/values/dimens.xml

**Review by iamareebjamal — DISMISSED** [review]

**Comment by iamareebjamal** [comment]
Resolve conflicts

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (iamareebjamal) [inline app/src/main/res/values/styles.xml]:
```suggestion
</resources>

```

**Comment by vishal-sehgal** [comment]
@iamareebjamal Resolved all conflicts

**Review by iamareebjamal — APPROVED** [review]
